install.sh

```diff
@@ -120,6 +120,42 @@ nvm_detect_profile() {
   fi
 }
 
+#
+# Check whether the user has any globally-installed npm modules in their system
+# Node, and warn them if so.
+#
+nvm_check_global_modules() {
+  local NPM_GLOBAL_MODULES
+  NPM_GLOBAL_MODULES=$(npm list -g --depth=0 | sed '/ npm@/d')
+  
+  local MODULE_COUNT
+  MODULE_COUNT=$(
```

Comment:
Does this work if `npm` isn't installed at the time of installation?

Please note that `install.sh` could be run a) when no node exists, b) when system node is active, c) when an nvm node is active.